html/css/404.css

```diff
@@ -0,0 +1,84 @@
+.fundo{
```

Comment:
Hình như phải như này mà c
```suggestion
.fundo {
```